indy_node/test/write_permission/test_revocation_write_permission.py

```diff
@@ -33,57 +24,55 @@ def tconf(tconf):
         make_state_path_for_claim_def(author_did,
                                       str(claim_def_req['operation'][CLAIM_DEF_SCHEMA_REF]),
                                       claim_def_req['operation'][CLAIM_DEF_SIGNATURE_TYPE],
                                       claim_def_req['operation'][CLAIM_DEF_TAG]
                                       ).decode()
-    revoc_req = sdk_sign_request_from_dict(looper, sdk_wallet_client, revoc_reg['operation'])
+    revoc_req = sdk_sign_request_from_dict(looper, sdk_wallet_trust_anchor, revoc_reg['operation'])
     _, revoc_reply = sdk_send_and_check([json.dumps(revoc_req)], looper, txnPoolNodeSet, sdk_pool_handle)[0]
     return revoc_req
 
 
-@pytest.fixture(scope="module")
-def trust_anchor_send_revoc_reg_def(looper,
-                                    txnPoolNodeSet,
-                                    sdk_wallet_trust_anchor,
-                                    sdk_pool_handle,
-                                    build_revoc_def_by_trust_anchor,
-                                    claim_def, tconf_false):
-    # We need to have claim_def to send revocation txns
+def test_client_cant_send_revoc_reg_def(looper,
+                                        txnPoolNodeSet,
+                                        sdk_wallet_client,
+                                        sdk_wallet_trust_anchor,
+                                        sdk_pool_handle,
+                                        build_revoc_def_by_client,
+                                        claim_def, tconf):
+    # when ANYONE_CAN_WRITE is false only trustee, steward, and trust_anchor can write
 
     claim_def_req = sdk_sign_request_from_dict(looper, sdk_wallet_trust_anchor, claim_def)
     sdk_send_and_check([json.dumps(claim_def_req)], looper, txnPoolNodeSet, sdk_pool_handle)
 
-    _, author_did = sdk_wallet_trust_anchor
-    revoc_reg = build_revoc_def_by_trust_anchor
+    _, author_did = sdk_wallet_client
+    revoc_reg = build_revoc_def_by_client
     revoc_reg['operation'][CRED_DEF_ID] = \
         make_state_path_for_claim_def(author_did,
                                       str(claim_def_req['operation'][CLAIM_DEF_SCHEMA_REF]),
                                       claim_def_req['operation'][CLAIM_DEF_SIGNATURE_TYPE],
                                       claim_def_req['operation'][CLAIM_DEF_TAG]
                                       ).decode()
-    revoc_req = sdk_sign_request_from_dict(looper, sdk_wallet_trust_anchor, revoc_reg['operation'])
-    _, revoc_reply = sdk_send_and_check([json.dumps(revoc_req)], looper, txnPoolNodeSet, sdk_pool_handle)[0]
-    return revoc_req
-
-
-def test_client_can_send_revoc_reg_def(client_send_revoc_reg_def):
-    pass
+    revoc_req = sdk_sign_request_from_dict(looper, sdk_wallet_client, revoc_reg['operation'])
+    with pytest.raises(RequestRejectedException):
+        _, revoc_reply = sdk_send_and_check([json.dumps(revoc_req)], looper, txnPoolNodeSet, sdk_pool_handle)[0]
 
```

Comment:
Also we need tests to check that only the one who created a RevocRegDef can create or edit corresponding RevocRegEntry.